project_mis_report/i18n/es.po

```diff
@@ -50,18 +57,27 @@ msgstr "Proyecto"
 
 #. module: project_mis_report
 #: code:addons/project_mis_report/models/project_project.py:0
+#: code:addons/project_mis_report/models/project_project.py:0
+#: code:addons/slv-mis-builder-2/project_mis_report/models/project_project.py:0
+#: code:addons/slv-mis-builder-4/project_mis_report/models/project_project.py:0
```

Comment:
Ouch!!! Soft link technique seems to have this issue generating translations. Could you remove these disturbing lines?